catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/CachingFederationStrategyTest.java

```diff
@@ -206,6 +207,30 @@ public void testFederateGetResults() throws Exception {
     assertThat(federateResponse.getResults().size(), is(sourceList.size()));
   }
 
+  @Test
+  public void testFederateLatencyMetrics() throws Exception {
+    QueryRequest fedQueryRequest = new QueryRequestImpl(mockQuery, false, null, properties);
+
+    when(mockResponse.getProperties()).thenReturn(properties);
+
+    Source mockSource1 = getMockSource();
+    Source mockSource2 = getMockSource();
+    Source mockSource3 = getMockSource();
+
+    List<Source> sourceList = ImmutableList.of(mockSource1, mockSource2, mockSource3);
+    QueryResponse federateResponse = federateStrategy.federate(sourceList, fedQueryRequest);
+
+    List<Map.Entry<String, Serializable>> metrics =
+            federateResponse
+                    .getProperties()
+                    .entrySet()
+                    .stream()
+                    .filter(entry -> entry.getKey().startsWith("metrics.source.elapsed."))
```

Comment:
same as above, consider using `SourceMetricsImpl.METRICS_SOURCE_ELAPSED` for `metrics.source.elapsed.` if available or consider moving this const to a common class.